@types/phaser3-rex-plugins/plugins/board/board/Board.d.ts

```diff
@@ -0,0 +1,122 @@
+declare module 'phaser3-rex-plugins/plugins/board/board/Board' {
+    export default class Board extends EE {
+        constructor(scene: any, config: any);
+        scene: any;
+        boardData: BoardData;
+        resetFromJSON(o: any): Board;
+        isBoard: any;
+        boot(): void;
+        shutdown(): Board;
+        destroy(): Board;
+        setGrid(grid: any): Board;
+        grid: any;
+        setWrapMode(mode: any): Board;
+        wrapMode: any;
+        setInfinityMode(mode: any): Board;
+        infinityMode: any;
+        setBoardSize(width: any, height: any): Board;
+        exists(gameObject: any): boolean;
+        get chessCount(): number | undefined;
+        clear(destroy: any): Board;
+        on(
+            event: string | symbol,
+            fn: import('phaser3-rex-plugins/node_modules/eventemitter3').ListenerFn,
+            context?: any,
+        ): Board;
+        addListener(
+            event: string | symbol,
+            fn: import('phaser3-rex-plugins/node_modules/eventemitter3').ListenerFn,
+            context?: any,
+        ): Board;
+        once(
+            event: string | symbol,
+            fn: import('phaser3-rex-plugins/node_modules/eventemitter3').ListenerFn,
+            context?: any,
+        ): Board;
+        removeListener(
+            event: string | symbol,
+            fn?: import('phaser3-rex-plugins/node_modules/eventemitter3').ListenerFn | undefined,
+            context?: any,
+            once?: boolean | undefined,
+        ): Board;
+        off(
+            event: string | symbol,
+            fn?: import('phaser3-rex-plugins/node_modules/eventemitter3').ListenerFn | undefined,
```

Comment:
Let's move this imports to top `import * as EventEmitter from 'eventemitter3'`
and use `EventEmitter. ListenerFn`